web/html/src/branding/css/base/responsive-rules.less

```diff
@@ -0,0 +1,31 @@
+@media (max-width: @screen-tablet) {
```

Comment:
This file is safe to drop, we don't include or build `.less` files anymore since we dropped the old theme. The `.scss` files are now the only source of styles.